cypress/integration/ui/create_dashboard/index.js

```diff
@@ -0,0 +1,111 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const UID_LENGTH = 11
+
+const dashboardTitle = new Date().toString()
```

Comment:
I think this is a pretty clever way to name the items created by the tests. You can use `toUTCString` or `toLocaleString` to get a more compact version (without the `GMT+0100 (Central European Standard Time)` part)

Reply:
Done